src/components/WorkOrders/JobsTable.js

```diff
@@ -38,6 +38,26 @@ const JobsTable = ({ workOrders }) => (
         ))}
       </tbody>
     </table>
+    <div className="page-navigation next-page-button govuk-!-padding-bottom-5">
+      {pageNumber > 1 && (
+        <button
+          className="govuk-button previous-page-button"
+          data-module="govuk-button"
+          onClick={() => handlePageClick(pageNumber - 1)}
+        >
+          Previous page
+        </button>
+      )}
+      {pageNumber && (
+        <button
+          className="govuk-button next-page-button"
+          data-module="govuk-button"
+          onClick={() => handlePageClick(pageNumber + 1)}
+        >
+          Next page
+        </button>
+      )}
+    </div>
```

Comment:
I'd have a look on https://hmcts-design-system.herokuapp.com/components/pagination about design for pagination. Not sure a button is the right thing to use - maybe can check in with a designer

Reply:
For now, we agreed that it will be a button next and previous only